rewrite-java/src/test/java/org/openrewrite/java/format/WrapMethodInvocationArgumentsTest.java

```diff
@@ -0,0 +1,862 @@
+                  void test() {
+                      Person p = new Person("John", 30, true);
+                  }
+              }
+              """,
+            """
+              package com.example;
+              
+              class Person {
+                  Person(String name, int age, boolean active) {}
+              }
+              
+              class Test {
+                  void test() {
+                      Person p = new Person(
+              "John",
+              30,
+              true);
+                  }
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void formatStaticMethodInvocation() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              package com.example;
+              
+              class Test {
+                  void test() {
+                      staticMethod("hello", 42, true);
+                  }
+              
+                  static void staticMethod(String s, int i, boolean b) {}
+              }
+              """,
+            """
+              package com.example;
+              
+              class Test {
+                  void test() {
+                      staticMethod(
+              "hello",
+              42,
+              true);
+                  }
+              
+                  static void staticMethod(String s, int i, boolean b) {}
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void formatMethodInvocationWithVarargs() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              package com.example;
+              
+              class Test {
+                  void test() {
+                      method("prefix", 1, 2, 3);
+                  }
+              
+                  void method(String prefix, int... values) {}
+              }
+              """,
+            """
+              package com.example;
+              
+              class Test {
+                  void test() {
+                      method(
+              "prefix",
+              1,
+              2,
+              3);
+                  }
+              
+                  void method(String prefix, int... values) {}
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void formatMethodInvocationWithComments() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              package com.example;
+              
+              class Test {
+                  void test() {
+                      method("hello", /* age */ 42, true);
+                  }
+              
+                  void method(String s, int i, boolean b) {}
+              }
+              """,
+            """
+              package com.example;
+              
+              class Test {
+                  void test() {
+                      method(
+              "hello", /* age */
+              42,
```

Comment:
When I ask IntelliJ to put the arguments on separate lines it produces the following. Should we follow that?
```suggestion
              "hello",
              /* age */ 42,
```